apps/core-v2-frontend/src/ui/app/Header.tsx

```diff
@@ -1,16 +1,18 @@
 import { ConnectButton } from "@rainbow-me/rainbowkit";
-import { ReactElement } from "react";
+import { ReactElement, useEffect, useState } from "react";
 
 export default function Header(): ReactElement {
+  // Hack to prevent Rainbow's ConnectButton from crashing during ssr
```

Comment:
I think using dynamic imports to prevent ssr for the button component is better since it's at the module level instead of react level.  can be done in a follow up

https://blog.bitsrc.io/using-non-ssr-friendly-components-with-next-js-916f38e8992c

Reply:
Great call, using `dynamic` instead